Templates/SaaSFileTrigger-FSharp/run.fsx

```diff
@@ -0,0 +1,3 @@
+let Run(input: string, name: string, output: byref<string>, log: TraceWriter) =
+    log.Info(sprintf "F# SaaS trigger function processed file: %s" name)
```

Comment:
You should be able to simplify this function by removing the byref parameter, using $return as the name for the binding in function.json, and simply returning input from this function.